src/add/observable/bindCallback.ts

```diff
@@ -3,4 +3,8 @@ import {BoundCallbackObservable} from '../../observable/BoundCallbackObservable'
 
 Observable.bindCallback = BoundCallbackObservable.create;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  module Observable {
```

Comment:
Yes please, if that's not too much trouble. The less overloaded terms I need to disambiguate in my head the better :smile: